src/Symfony/Bundle/SecurityBundle/Templating/Helper/LogoutUrlHelper.php

```diff
@@ -95,8 +99,16 @@ public function getLogoutUrl($key)
      *
      * @throws \InvalidArgumentException if no LogoutListener is registered for the key
      */
-    private function generateLogoutUrl($key, $referenceType)
+    private function generateLogoutUrl($key = null, $referenceType)
```

Comment:
no need to add a default value here: 

- it is not optional anyway because there is a mandatory argument after it
- the argument is always passed explicitly to this private method anyway